src/Widgets/CameraView.vala

```diff
@@ -205,7 +205,7 @@ public class Camera.Widgets.CameraView : Gtk.Stack {
         var camera = find_camera_from_name (display_name);
         try {
             if (camera == null) {
-                throw new IOError.NOT_FOUND (_("%s could not be found"), display_name);
+                throw new IOError.NOT_FOUND (_("Could not find %s"), display_name);
```

Comment:
As long as we’re fixing this, we should quote the name by convention.

```suggestion
                throw new IOError.NOT_FOUND (_("Could not find “%s”"), display_name);
```

Reply:
Depends what it is, but yes.